kyuubi-server/src/test/scala/org/apache/kyuubi/events/EventLoggingServiceSuite.scala

```diff
@@ -149,4 +154,76 @@ class EventLoggingServiceSuite extends WithKyuubiServer with HiveJDBCTestHelper
       }
     }
   }
+
+  test("test Kyuubi server info event") {
+    val confKv = List(("awesome.kyuubi", "true"), ("awesome.kyuubi.server", "yeah"))
+    for (kv <- confKv) {
+      conf.set(kv._1, kv._2)
+    }
+    val name = "KyuubiServerInfoTest"
+    val server = new KyuubiServer(name)
+    server.initialize(conf)
+    server.start()
+
+    val hostName = InetAddress.getLocalHost.getCanonicalHostName
+    val kyuubiServerStartEventPath =
+      Paths.get(serverLogRoot, "kyuubi_server_info", s"day=$currentDate", s"server-$hostName.json")
+
+    def compareResultWithState(state: String, result: ResultSet): Unit = {
+      result.next()
+      assert(result.getString("serverName") == name)
+      result.next()
```

Comment:
why we need so much `result.next()` here ? isn't one kyuubi server only contains two rows of kyuubi server event ?

Reply:
No, when we start or stop kyuubi server normally, it will create an event. 

Default partitions function is `yyyyMMdd`. If when we config `backend.server.event.loggers` is `JSON`, that means the JSON file will have more than one or two events, it depends on as we start or stop kyuubi server in one day.

In this test, when we use SQL to query the JSON file, one row is one event. So I use result.next() to get every column in one row result.

Reply:
One row, one event. When we query one row as `ResultSet`, we need to use `result.next()` to get next col in the `ResultSet`.